src/general.c

```diff
@@ -227,28 +227,30 @@ gdip_get_display_dpi ()
 GpPointF *
 convert_points (const GpPoint *point, int count)
 {
-	int i;
+	if (!point || count < 0)
```

Comment:
I'd suggest `count <= 0` since it would return NULL anyway in the next step.

Reply:
Can't change this - on my PC this causes test failures in `testpathgradientbrush`

Will investigate further in another PR?

Reply:
That's weird, but I am fine with postponing it.